app/src/main/java/io/techery/properratingbarsample/MainActivity.java

```diff
@@ -43,7 +43,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private RatingListener ratingListener = new RatingListener() {
         @Override
-        public void onRatePicked(int rating) {
+        public void onRatePicked(ProperRatingBar ratingBar) {
```

Comment:
Why? We can call ratingBar.getRating() (or whatever other methods will appear for ProperRatingBar instance later) on it.